fastlane/Fastfile

```diff
@@ -25,10 +23,8 @@ platform :android do
       json_key: "playstore-secret.json",
       apk_paths: [
         "build/outputs/apk/#{options[:flavor]}Webview/release/cht-android-#{version}-#{options[:flavor]}-webview-arm64-v8a-release.apk",
-        # NOT SUPPORTED WITH: targetSdkVersion 29. Need to add it back to support older Android versions
-        # "build/outputs/apk/#{options[:flavor]}Webview/release/cht-android-#{version}-#{options[:flavor]}-webview-armeabi-v7a-release.apk",
-        "build/outputs/apk/#{options[:flavor]}Xwalk/release/cht-android-#{version}-#{options[:flavor]}-xwalk-arm64-v8a-release.apk",
-        "build/outputs/apk/#{options[:flavor]}Xwalk/release/cht-android-#{version}-#{options[:flavor]}-xwalk-armeabi-v7a-release.apk",
+        # Support for older versions of Android:
+        "build/outputs/apk/#{options[:flavor]}Webview/release/cht-android-#{version}-#{options[:flavor]}-webview-armeabi-v7a-release.apk",
```

Comment:
While we're here I think we should remove every mention of "webview" from the apk names, and the gradle tasks and so on. We should also merge `src/webview` into `src/main` because there's no point having the two separate any more!

Reply:
Oh great catch! Working on it!